fl4health/server/base_server.py

```diff
@@ -350,3 +350,69 @@ def _hydrate_model_for_checkpointing(self) -> nn.Module:
         model_ndarrays = parameters_to_ndarrays(self.parameters)
         self.parameter_exchanger.pull_parameters(model_ndarrays, self.server_model)
         return self.server_model
+
+
+class FlServerWithInitializer(FlServer):
+    initialized = False  # Add attribute
+
+    def _get_initial_parameters(self, server_round: int, timeout: Optional[float]) -> Parameters:
+        """
+        Get initial parameters from one of the available clients. Same as
+        parent function except we provide a config to the client when
+        requesting initial parameters
+        https://github.com/adap/flower/issues/3770
+
+        Note:
+            I have to use configure_fit to bypass mypy errors since
+            on_fit_config is not defined in the Strategy base class. The
+            downside is that configure fit will wait until enough clients for
+            training are present instead of just sampling one client. I
+            thought about defining a new init_config attribute but this
```

Comment:
Thanks for the additional context! It seems like you forgot to finish the thought at the end though